src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -82,10 +80,7 @@
 public class ExperimentProcessor implements Processor {
 
     private static final String NAME = "Experiment";
-    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experiment Units are missing Observation Unit Id.<br/><br/>" +
-            "If you’re trying to add these units to the experiment, please create a new environment" +
-            " with all appropriate experiment units (NOTE: this will generate new Observation Unit Ids " +
-            "for each experiment unit).";
+    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experimental entities are missing ObsUnitIDs. You must append file with the appropriate ObsUnitIDs or specify a new environment for these entities to proceed.";
```

Comment:
The source of truth reads,`Error detected in file, XXX.xls.  Experimental entities are missing ObsUnitIDs. Import cannot proceed.`

The current error message reads,`Error(s) detected in file, XXX.xls. Experimental entities are missing ObsUnitIDs. You must append file with the appropriate ObsUnitIDs or specify a new environment for these entities to proceed.. Import cannot proceed. `

I recommend changing MISSING_OBS_UNIT_ID_ERROR to "Experimental entities are missing ObsUnitIDs"

NOTE: there is no period (.) at the end of this string.  It is added by the javascript code.


[ I do like your wording, at the very least, please change  MISSING_OBS_UNIT_ID_ERROR to "Experimental entities are missing ObsUnitIDs. You must append file with the appropriate ObsUnitIDs or specify a new environment" ( and leave off " for these entities to proceed." ) ]